### ISSUE TITLE
story label bios

### ISSUE THREAD

**Issue by JustMeSaying** [newsvoice/newsvoice#208]
Allow reader to vote if individual stories have a political bios, or push a political agenda, or if they do a good job of just presenting facts in a clear, accurate manner.

AKA crowdsource a label for individual stories

**Comment by frog712** [comment]
This is a great idea, however there would need to be a percentage threshold before the label was added to prevent abuse.

**Comment by vlidholt** [comment]
Great idea. Will try to get this done in some shape or form!

**Comment by JustMeSaying** [comment]
After giving it more thought its not such a great idea for the following reasons, put in as simple and abrupt manner as possible.
1. It puts to much faith in real individuals creating responses independently.  
2. You can't afford the kind of review process it would take to eliminate bot controlled responses.

You can afford to organize, provide simple tools and training a few groups of broad subject expert bios flaggers / editorial reviewers.
Their primary function would be to identify and flag overtly bios, misleading/manipulative with intent, and fabricated events or statements.
They should not have the ability or authority to remove any post, except in extreme cases. In fact no user posts should ever be removed except in extreme cases.

The subject expert comes in because it is not enough to flag a post as bios in an agenda driven manner. The teams must be able to go in highlight sections of bios in the original post and state why it is bios and provide the objective of the bioses, how harmful it is, who it hurts or helps etc.

Simply put the goal is to INDENTIFY, EDUCATE, and SHAME the practice out of existence in .

Now back full circle. These groups can be over over ruled by the public and the original content allowed to stand unmolested by supermajority vote. 
This I think like impeachment would be rare.

**Comment by Digiguy** [comment]
So what you are talking about is democracy by representation?
Yes that works well, and I believe the system now of by points is a good way to show engagement as a qualification to apply. 
Second would be the ability to critically think and judge without bias. 
Third would be the body of the comments and contributions to that point. (IE 10,000 crap posts do not equal the right to be the mod as you mention.
Lastly due to recent conversations, those moving beyond simple users should be identity verified with a LinkedIn profile to only 1 member of the administration. This gives some credence to the seriousness and actual accuracy of who is wanting to be part of the team. "With responsibility comes proof of ability" ... or in this case, verified identity and sharing of trust to achieve team membership. There is also the fact that people need to be held accountable for their actions and those without a face are beholden to no one. Total anonymity does no bode well for being responsible.

I believe in a voting system/ polling system method of reference gathering. 
There can be a selection of a range for a story without an AI generated type of list even based on the content. Each vote gets them a point as well towards the gamification aspect of the app. Badges, member perks, app add-ons, etc to encourage participation for the group.

1 - 10 pull pop up range slider for the following choices. (quick and easy to do)
Well written, detailed, referenced, opinion based.
Those type of choices give the reader then muliti-bar displayed to them of vote totals once they participate. (like youtube polls)

**Comment by JustMeSaying** [comment]
A point based system has proven critical flaws that not YouTube, FaceBook, Disqus, or Google have been able to demonstrate they can overcome. I hope you're not suggesting a relatively small startup like Newsvoice can.
A poorly informed, mislead, emotionally manipulated voter base and mechanizim tampering deliberate or otherwise is the entire reason a representative democracy can work and a direct one never will.
To judge applicants abilities to critically think, analyze, research, and publish, while maintaining complete separation of any personal bios (real or imagined) in a work environment on past social media content is naive, and does not bode well for the selection process.
Reliance on LinkedIn as a soul source for accuracy, accountability, and proof of ability is NEED I SAY IT... discriminatory, limiting, and just plain stupid.
With a serious application process in place that does not exist yet on Newsvoice, and formal submission comes the confidentially privileged reveal of any applicants identity, history, and background, until then its nobodies business, and it should not ever be requested. Businesses and individuals repeatedly demonstrate an inability to properly care for, secure, and not misuse for personal gain such information. 
Liability much of the time is about process, and not having or following legally accepted process standards has sunk more than one or two startups, my suggestion is you start paying attention to process, like yesterday.

Any content publisher and its moderators, editors etc should always operate with full and complete transparency in both selection process and personal related backgrounds. That should be an operational ethical standard for the publishing industry that is a given, does not require stating, and if not followed is grounds for immediate termination.

Gamification of the app are you serious? That is a total violation of the ethical standards of the publication industry, any serious journalist, and not just an open door but and invitation to corruption, malfeasance, and much much more. To even suggest it disqualifies you by any serious critical thinking standard metric.
It introduces a entirely new incentive based, digital media propaganda pushing content publisher as a business model, wtf. 
It is completely contrary to Newsvoices stated goals, violates multiple federal laws, it incentives crimes against the Constitution by attacking various creations of and, their ongoing, protected processes held within it. Who are you Donald Trump?
You clearly have never had any course work in business ethics, HR employment standards & practices, financial or administrative operations, or basic governmental civics course work. Get some or you will make some lawyers a lot of money one day and possibly end up in a country club prison, for any number of "white collar" crimes including acts of treason which such an attack your, as outlined incentivised propaganda machine would invite upon the Constitution, and all forms of democratically elected governments globally.
Sadly you are more representative than not of your particular demographic, very sad indeed.

**Comment by Digiguy** [comment]
I had a long answer, going into detail how you are condescending and threatening in attitude. Incorrect in so many ways it gave me a headache trying to see how someone could be so condescending.... but I did get you are American, it did explain a lot. Boomer probably too. I decided ya, .. who was it said... "some men you just cannot reach".... (Hint I know who and what he meant)

 You seem to think you have all the answers, but obviously..
"The ignorant man says he knows best, the wise man knows best not to say."

I for one, am Canadian though. You know, the country with real equal rights? And hate crime laws?
Sadly.... well .... read what you wrote. 
Projection is a bitch.

**Comment by Digiguy** [comment]
OH... have a nice evening. It was terrific having you talk AT me.
I know in no way would you listen to any opinion or proof of my ideas I raised, so I took the high road and left you to your own devices.
Good day.

**Comment by JustMeSaying** [comment]
You're actually going to publicly state YouTube, FaceBook, Disqus, or
Google are not almost being continually hacked, poll results and stories
manipulated, created, and deleted, and user data stolen, all from outside
their secure systems?
That appears to be what you are saying, so before calling you something far
beyond stupid, I ask you to verify that is your position statement.

The ethics and morality of the gamification and incentivication of a web
based content publisher, with a stated goal of fighting inaccurate, false,
ideologically based propagandist content, well that's on you.

Sorry you got butthurt,

Signed
Dickish

**Comment by JustMeSaying** [comment]
Unbunch those panties sweetheart.

**Comment by Digiguy** [comment]
I am not commenting to you at all.
Repeat your opinions till the cows come home. 
You don't respect me, why would I talk to you?
I, quite frankly, am better than that.
I want no conversation with the likes of you.
Insults and homophobic innuendo will get you no where.
Have a nice day 'Dickish'.. that somehow appears to be honest. Well done see the self reflection is working.

**Comment by JustMeSaying** [comment]
I am completely at ease with and confident in the public deciding if my assessment of your stated plan is wildly off the mark or spot on.
They can also judge my direct, if brutish criticisms of your policy positions against your unsupported, rather whiny, self absorbed but not self aware, wannabe elitist, attacks against me personally.
Why is that, is t because you are unable to support or defend your positions, so instead you play the you hurt my feelings, everyone feel sorry for me game, then you cut & run.

Are you really as you said "better than that"
If you got homophobic from the singular word butthurt, then you also have no understanding of what real homophobia is or how it feels. Try growing up and being openly gay in a world before Pricilla Queen of the Desert, Ellen, Will & Grace, or any real legal protections even in your so much better than everywhere else Canada. How many marches, fundraisers, and protests have you been in? Do you remember Act Up or even know what it is? 
Was your fain indignation just a bs act for sympathy, because now you have made me angry.
Shut the F up and sit down boy.
Grow up homophobia ends when you don't get upset at being called one, but are proud of all that you are, and openly accept it as a complement regardless of who said it or its intent.
If you don't own the power it will be taken from you. Explore that thought will ya.
I'm done

**Comment by Digiguy** [comment]
my my Such anger. And the language! good grief!
Finally you are done? really? 
Are you sure? 
Why do I feel you will sit and stew. and write more... Karnak says.... "lol"
I was wondering when the sermon of how much you think you know would end.

**Comment by frog712** [comment]
@Digiguy if you're just going to be passive-aggressive rather than provide an actual argument, get off your soapbox and leave
>I, quite frankly, am better than that.
>I want no conversation with the likes of you.

a true intellectual™

**Comment by Digiguy** [comment]
frog712.... had a long answer, going into detail how you are condescending and threatening in attitude. Incorrect in so many ways it gave me a headache trying to see how someone could be so condescending.... but I did get you are American, it did explain a lot. Boomer probably too. I decided ya, .. who was it said... "some men you just cannot reach".... (Hint I know who and what he meant)

You seem to think you have all the answers, but obviously..
"The ignorant man says he knows best, the wise man knows best not to say."

I for one, am Canadian though. You know, the country with real equal rights? And hate crime laws?
Sadly.... well .... read what you wrote.
Projection is a bitch.

**Comment by JustMeSaying** [comment]
To everyone else, sorry for going off like that, I apologies. 
Sometimes when you were in the fight for just the basics. Basics like not having to search for hidden little symbols that said in a kind of code this is a safe place for you, and your community to shop, or stay, or even have a meal with your partner, and someone acts like they know the struggle and the fight it can set you off.
Yes gays and lesbians had their own "green book" of sorts and it wasn't all that long ago.

**Comment by Digiguy** [comment]
Again,
You lost your chance at a conversation with all that at the beginning.
I have no need for people like you in my life at any point.
Apolgy accepted. Now stop talking to me.

**Comment by JustMeSaying** [comment]
> Projection is a bitch.
Digiguy if thats the projection you are looking for I'm sure you will find it if you just look in a mirror. Its probably something you should do for a long hard time.
